airbyte-webapp/src/components/EntityTable/components/ConnectionSettingsCell.tsx

```diff
@@ -39,7 +38,7 @@ const ConnectorCell: React.FC<IProps> = ({ id }) => {
     event.stopPropagation();
   };
 
-  const settingPath = `/${RoutePaths.Workspaces}/${workspaceId}/${RoutePaths.Connections}/${id}/${ConnectionSettingsRoutes.REPLICATION}`;
+  const settingPath = `/${RoutePaths.Workspaces}/${workspaceId}/${RoutePaths.Connections}/${id}/${ConnectionPageRoutePaths.REPLICATION}`;
```

Comment:
nitpick: could we use a uniform capitalization of enums for the paths? e.g. change to`ConnectionPageRoutePaths.Replication` or `RoutePaths.WORKSPACES` and `RoutePaths.CONNECTIONS`

Reply:
Hmm, it looks like we have inconsistent conventions across the app. I'll follow up to see what our convention should be.